### PR TITLE
fix: add global footer to subpages

### PR DESCRIPTION
**Pull request by wmui51**
# READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
- TL;DR - what's this PR for?
- Endowed Positions subpages were missing the global footer, this should add to all schools/centers pages along with the search results page.

# Review By (Date)
- When does this need to be reviewed by?
- @forrestglick can provide details for how urgent this is.

# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Review Tasks

## Setup tasks and/or behavior to test

1. Go to preview link and append /endowed-positions/graduate-school-of-business
2. Confirm the global footer is now there
3. Try a search term
4. Confirm the global footer is on the search results page

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

**Review by yvonnetangsu — APPROVED** [review]
I do see the search result pages having the global footer
https://deploy-preview-354--adapt-giving.netlify.app/endowed-positions/search?term=business

The links from the endowed-positions page don't but I think that's because they're linked to the dev branch preview site which doesn't have this fix yet.

Approving this and I'll add this to the release tomorrow.
